prime-router/docs/getting-started/install-ase.md

```diff
@@ -0,0 +1,56 @@
+# Installing Azure Storage Explorer
```

Comment:
I honestly think you do not need this file.  The instructions are probably available at the MS site.

Reply:
Agreed, linking to the MS site directly